packages/bulkapi/src/BulkApiContext.ts

```diff
@@ -32,6 +32,8 @@ export function registerBulkApi(app: Express, dbConnection: pgPromise.IDatabase<
     const context = new BulkApiContext(dbConnection, pgp)
     
     // Add routes to application
+    app.post("/bulk/createPartitions", runWithTry(context.bulkApi.createPartitions))
```

Comment:
It would be convenient to have a way to check if a partition with a given id already exists. One could use `/partitions` asking anche check in the return list, but in case there are many partitions it could unneccessarily expensive, I suppose

Reply:
One could also call _retrieve_  with the id of the partition, and a depth of 0. That should not give too much overhead.